voorhoede/dda-website
# Feat: Fonts optimizations

# Changes

- Replace fonts with webfont subsets

# Associated issue

Resolves https://trello.com/c/53ypglX4/39-fonts-optimaliseren

# How to test

1. Open preview link
2. Check the inspects network tab
3. Filter for fonts
4. Check if woff2 is fetched

# Checklist
